R/col_plan.R

```diff
@@ -0,0 +1,497 @@
+#' Define the Column Plan
```

Comment:
This is very thorough but is currently moreso than other "plans" in the package. I would love for them to be more consistent in terms of checking, classes, etc. But probably something that can be tabled for later!